crates/runc/src/lib.rs

```diff
@@ -368,6 +368,22 @@ pub trait Spawner: Debug {
 impl Runc {
     async fn launch(&self, cmd: Command, combined_output: bool) -> Result<Response> {
         debug!("Execute command {:?}", cmd);
+        #[cfg(target_os = "linux")]
+        let mut cmd = cmd;
```

Comment:
Since `runc` is Linux only, we can rewrite this to something like:

```rust
    async fn launch(&self, mut cmd: Command, combined_output: bool) -> Result<Response> {
        debug!("Execute command {:?}", cmd);

        if let Ok(thp) = std::env::var("THP_DISABLED") {
            if let Ok(thp_disabled) = thp.parse::<bool>() {
                unsafe {
                    cmd.pre_exec(move || {
                        #[cfg(target_os = "linux")]
                        if let Err(e) = prctl::set_thp_disable(thp_disabled) {
                            log::debug!("set_thp_disable err: {}", e);
                        };

                        Ok(())
                    });
                }
            }
        }
```